.github/workflows/copilot-deploy-binaries.yml

```diff
@@ -52,7 +40,6 @@ jobs:
       AZURE_SUBSCRIPTION_ID: ${{secrets.AZURE_SUBSCRIPTION_ID}}
 
   deploy-frontend:
-    needs: [deploy-infra]
     uses: ./.github/workflows/copilot-deploy-frontend.yml
     with:
       DEPLOYMENT_NAME: ${{needs.deploy-infra.outputs.deployment-id}}
```

Comment:
This won't work without having deploy-infra in the needs collection. `{{ needs.deploy-infra...`

Reply:
Currently the front-end and back-end deployments use the deployment Id to get some information about the resources so they need a reference to it. In theory you could pass this in as a parameter. However, I'm not sure we want to do this before the build has succeeded.

Reply:
Argh. Of course!  Whst did I tell you about GH's pipeline syntax...

Abandonning this PR